Src/LiquidProjections/EventMapBuilder.cs

```diff
@@ -69,6 +80,11 @@ public Action(EventMapBuilder<TContext> parent, Func<ProjectorMap<TContext>> get
             {
                 this.parent = parent;
                 this.getProjector = getProjector;
+
+                foreach (Func<object,TContext,Task<bool>> predicate in parent.predicates)

```

Comment:
This way the map-level filters only apply to event handlers registered after the filters are registered.